ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/RDXVisualizer.java

```diff
@@ -64,10 +64,6 @@ public ROS2Heartbeat getHeartbeat()
    public void create()
    {
       createdYet = true;
-      if (getPanel() != null)
-      {
-         setActive(getPanel().getIsShowing().get());
-      }
```

Comment:
This was problematic, as it was disabling the visualizers even though the user setActive(true) during construction.